src/js/node/child_process.ts

```diff
@@ -1621,6 +1621,10 @@ function validateString(value, name) {
   if (typeof value !== "string") throw new ERR_INVALID_ARG_TYPE(name, "string", value);
 }
 
+function isInt32(value) {
```

Comment:
is this a builtin? I think it's a builtin

Reply:
https://github.com/nodejs/node/blob/c1d9f426a11eda97/lib/internal/validators.js#L39-L45